.github/workflows/coverage.yml

```diff
@@ -2,7 +2,7 @@ name: CodeCov
 
 on:
   push:
-    branches: [ main, master ]
+    branches: [ main, master, packaging_and_versioning ]
```

Comment:
this change should probably be reverted before merging since it doesn't really make sense afterwards.

Reply:
done